### PR TITLE
Partially incorporate suggestions by @xloem and @ldz1, fix generator bug.

### PR DESCRIPTION
**Pull request by evoskuil**

### CODE REVIEW

File: gsl.copy_properties.sh [hunk 1]
```diff
@@ -72,9 +72,9 @@ function generate_artifacts(path_prefix)
         echo(" Evaluating repository: $(_repository.name)")
         for _repository->install.build as _build where\
             defined(_build.repository) & starts_with(_build.repository, "libbitcoin")
-            emit_import_copy(_repository, my.path_prefix, _build.repository)
+            emit_import_copy(_build.repository, my.path_prefix, _build.repository)
         endfor
-        emit_project_props_copy(_repository, my.path_prefix)
+        emit_project_props_copy(_build.repository, my.path_prefix)
```
Comment (ldz1) [inline gsl.copy_properties.sh:77]: _build doesn't exist at this point => undefined.

File: gsl.copy_properties.sh [hunk 2]
```diff
@@ -72,9 +72,9 @@ function generate_artifacts(path_prefix)
         echo(" Evaluating repository: $(_repository.name)")
         for _repository->install.build as _build where\
             defined(_build.repository) & starts_with(_build.repository, "libbitcoin")
-            emit_import_copy(_repository, my.path_prefix, _build.repository)
+            emit_import_copy(_build.repository, my.path_prefix, _build.repository)
```
Comment (ldz1) [inline gsl.copy_properties.sh:75]: _repository seems to be a node, _build.repository is a attribute value. We try to extract data from the node inside "emit_import_copyt" and it fails.

Comment (evoskuil) [reply gsl.copy_properties.sh:75]: Looks like my generation was succeeding only due to residual files.

**Comment by evoskuil** [comment]
Fixed up by https://github.com/libbitcoin/libbitcoin-build/pull/233

**Pull request merged**